packages/core/src/api/schema/admin-api/order-states.api.graphql

```diff
@@ -0,0 +1,3 @@
+type Query {
+    orderStates: [OrderState!]
```

Comment:
We already have this info available in the Admin API via the `globalSettings` query, which can return:

```ts
{
  serverConfig: {
    orderProcess: [ ... states ]
  }
}
```

So you can remove this and use that existing query in your Admin UI changes.

Reply:
I found it easier to make new PR: https://github.com/vendure-ecommerce/vendure/pull/514
closing this one